app/utils/certificates.py

```diff
@@ -57,7 +57,8 @@
     "MormorFärg": "MM_color",
     "Nummer": "number",
     "IntygasAv": "user_id",  # Username
-    "IntygasNummer": "certificate",  # Herd id of rabbit individual (Without herd identifier) G/M
+    # Herd id of rabbit individual (Without herd identifier) G/M
+    "IntygasNummer": "certificate",
```

Comment:
What was the problem here? Is it because the comment was to long to be in the same line?